src/ray/protobuf/autoscaler.proto

```diff
@@ -317,11 +317,32 @@ message DrainNodeReply {
   string rejection_reason_message = 2;
 }
 
+message NodeGroupConfig {
+  map<string, double> resources = 1;
```

Comment:
https://github.com/anyscale/product/blob/d27096d7b38faf091b3dd6a870c3d06877336c19/go/infra/proto/cluster_types.proto#L117
The resource we define is `map<string, int64> logical_resources = 6;`

Reply:
Ray uses `double` for resource quantities. I'll be consistent with Ray.